### ISSUE TITLE
Choose lowest of coinBalance, collateral, or tokenCollateral for image rendering

### ISSUE THREAD

**Issue by cupOJoseph** [open-dollar/od-contracts#560]
Dependent on #534 

```solidity
_renderCollatAndDebt(
              ratio, params.stabilityFee, params.debt, params.collateral, params.symbol, params.lastUpdate //tokenCollateral and coinBalance are not included here because they will usually be the same.
            ),
```

another addition here could be in the case where an internal transfer has changed the balance of one of these, display which ever is the lowest to the user.

As part of this issue, please also confirm a situation when these three variables are not the same.

**Comment by pi0neerpat** [comment]
No longer relevant, since we are not displaying internal balances
